### ISSUE TITLE
Sign text clears when rejoining

### ISSUE THREAD

**Issue by Bin-Ban** [Eggs-D-Studios/wos-issues#499]
### Guidelines

- [X] I checked for duplicate bug reports
- [X] I tried to find a way to reproduce the bug

### Version

Main (Production)

### What happened? What did you expect to happen?

When using a sign and assigning it a text, it works perfectly fine until I re-enter that region either by rejoining or warping to it. The text clears from the sign, yet when entiering the configuration of the sign, the text is there, and when clicking apply will appear again

### How can we reproduce this bug?

1. Make a sign and apply it text
2. Rejoin
3. Obverse sign

### Frequency

Always

**Comment by NameTemplate** [comment]
it's not cleared. check the config.

**Comment by raulaltillo** [comment]
yes I'm not saying the config is cleared, the sign text itself is

**Comment by Bin-Ban** [comment]
yes as I said, the config isn't cleared, it's there, but the text display itself isn't there unless you apply the text again

**Comment by Kajto359** [comment]
Never got fixed ):

**Comment by Hexcede** [comment]
Should now be fixed, let me know if you encounter any problems
